LLama/Native/NativeApi.LLava.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace LLama.Native;
+
+using clip_ctx = IntPtr;
+public static unsafe partial class NativeApi
+{
+    
+    /*/// <summary>
+    /// Clip Vision Parameters
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct clip_vision_hparams
+    {
+        public Int32 image_size;
+        public Int32 patch_size;
+        public Int32 hidden_size;
+        public Int32 n_intermediate;
+        public Int32 projection_dim;
+        public Int32 n_head;
+        public Int32 n_layer;
+        public float eps;
+    };*/
+
+    /// <summary>
+    /// LLaVa Image embeddings 
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct llava_image_embed
+    {
+        public float* embed;
+        public int n_image_pos;
+    }
+
+    /// <summary>
+    /// Load MULTI MODAL PROJECTIONS model / Clip Model
+    /// </summary>
+    /// <param name="mmProj"> Model path/file</param>
+    /// <param name="verbosity">Verbosity level</param>
+    /// <returns></returns>
+    [DllImport(llavaLibraryName, EntryPoint = "clip_model_load", CallingConvention = CallingConvention.Cdecl)]
+    public static extern clip_ctx clip_model_load(string mmProj, int verbosity);
+
+    /// <summary>
+    /// Frees MULTI MODAL PROJECTIONS model / Clip Model
+    /// </summary>
+    /// <param name="ctx"></param>
+    [DllImport(llavaLibraryName, EntryPoint = "clip_free", CallingConvention = CallingConvention.Cdecl)]
+    public static extern void clip_free(clip_ctx ctx);
+
+  
+    /// <summary>
+    /// Sanity check for clip <-> llava embed size match
```

Comment:
Lots of XML errors here, I tink it's because you've used `<->` instead of `&lt;-&gt;`